isc.class.php

```diff
@@ -210,12 +210,12 @@ public function get_image_by_url($url = '')
             }
             $types = implode( '|', $this->_allowedExtensions );
             /**
-             * check for the format 'image-title-(e12452112-)300x200.jpg(?query…)' and removes
+             * check for the format 'image-title-300x200.jpg(?query…)' and removes
```

Comment:
@lippoliv does your change not recognize the url in the comment anymore? These were created when you edit images (e.g. crop, resize) through WordPress.